src/components/calendar/calendar.css

```
@@ -37,6 +38,19 @@
   padding-top: var(--spacing-medium);
 }
 
+.calendar-year-list {
+  height: calc(var(--spacing-x-large) * 7); /* 7 list options */
+  padding-left: var(--spacing-medium);
+  margin-top: calc(0 - var(--spacing-medium));
+  margin-left: calc(var(--spacing-x-large) - var(--spacing-medium));
+  margin-right: calc(var(--spacing-x-large) - var(--spacing-medium));
+  margin-bottom: var(--spacing-medium);
+  overflow: scroll;
```

```suggestion
  overflow-y: scroll;
```

Otherwise, there is a horizontal scrollbar always visible (on non-macs)